project.clj

```diff
@@ -3,11 +3,38 @@
   :url "https://github.com/weavejester/clout"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.5.1"]
-                 [instaparse "1.4.0" :exclusions [org.clojure/clojure]]]
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/clojurescript "1.10.191" :scope "provided"]
+                 [instaparse "1.4.8" :exclusions [org.clojure/clojure]]]
+  :plugins [[codox "0.6.4"]
+            [lein-doo "0.1.7"]
+            [lein-npm "0.6.2"]
+            [lein-cljsbuild "1.1.4"]]
   :profiles
   {:dev {:jvm-opts ^:replace []
          :dependencies [[ring/ring-mock "0.2.0"]
-                        [criterium "0.4.2"]]}
-   :1.6 {:dependencies [[org.clojure/clojure "1.6.0"]]}
-   :1.7 {:dependencies [[org.clojure/clojure "1.7.0-beta2"]]}})
+                        [criterium "0.4.2"]]
+         :cljsbuild {:builds
```

Comment:
The `:cljsbuild` and `:doo` keys can be in the base project config, I think. There's nothing that can affect deployment, after all.